app/adapters/config.js

```diff
@@ -4,10 +4,10 @@ import Ember from 'ember';
 import PouchAdapterUtils from 'hospitalrun/mixins/pouch-adapter-utils';
 
 export default Adapter.extend(PouchAdapterUtils, {
-    databaseName: 'config',    
-    db:  Ember.computed.alias('pouchDBService.configDB'),
-    pouchDBService: Ember.inject.service('pouchdb'),
-    
+    databaseName: 'config',
+    database: Ember.inject.service(),
+    db:  Ember.computed.alias('database.configDB'),
```

Comment:
doesnt seem to be the case https://github.com/HospitalRun/hospitalrun-frontend/blob/master/app/mixins/pouch-adapter-utils.js